### DISCUSSION TITLE
支持 postcss 下可以支持 shadow-dom 吗？

### DISCUSSION THREAD

**Discussion by Ron-Dong** [unocss/unocss#3449] [Q&A]
vite项目现在可以，用rspack建的项目有办法支持 lit 框架吗？

**Comment by yoyo837** [comment]
同问，希望 postcss 插件方式的配置，能支持shadow-dom模式

**Reply by yoyo837** [reply]
我自己都忘记了自己创建过相同的话题 https://github.com/unocss/unocss/discussions/2406

**Reply by yoyo837** [reply]
https://github.com/unocss/unocss/discussions/2406#discussioncomment-9699776

`unocss/vite` 当前版本我试了支持`rollup`, 然后再给`rollup`启用`postcss`插件就可以了。
